src/plugins/content/accessibility/keyNavigation/strategies/navigatorNavigation.js

```diff
@@ -63,7 +63,13 @@ export default {
                     propagateTab: false,
                     elements: navigableFilters,
                     group: $navigator.find('.qti-navigator-filters')
-                });
+                })
+                    .on('focus', cursor => {
```

Comment:
I guess the key navigator should not care about handling events related to target elements. It should be their own responsibility.

Reply:
But we already have this approach here
https://github.com/oat-sa/tao-test-runner-qti-fe/blob/f28f17b3158531668365238c3f2e05f27a99a6f3/src/plugins/content/accessibility/keyNavigation/strategies/navigatorNavigation.js#L75
https://github.com/oat-sa/tao-test-runner-qti-fe/blob/f28f17b3158531668365238c3f2e05f27a99a6f3/src/plugins/content/accessibility/keyNavigation/strategies/navigatorNavigation.js#L139

Reply:
The handlers in your examples are related to the key navigator behavior itself. Anyway you should keep target elements logic in the  appropriate modules, if it possible